### PR TITLE
docs: guide to write adapters code

### PR DESCRIPTION
**Pull request by MicBun**
resolves: https://github.com/trufnetwork/truf-network/issues/55

### CODE REVIEW

File: writing-adapters.mdx [hunk 1]
```diff
@@ -0,0 +1,97 @@
+---
+title: Adapter Writing Guide
+description: This guide provides a step-by-step approach to creating adapters for the Truf Network (TN). Adapters facilitate the ingestion of data from various sources into TN, leveraging the SDK and Prefect for efficient data management.
+---
+
+# Developing Adapters for the Truf Network
+
+## Prerequisites
+
+- **Programming Knowledge**: Proficiency in Python.
```
Comment (markholdex) [inline writing-adapters.mdx:10]: @MicBun can adapters be written in other languages?

Comment (MicBun) [reply writing-adapters.mdx:10]: Yes, it is actually possible to be written in Go and Js. The example is given in Python. I will revise it.

File: writing-adapters.mdx [hunk 2]
```diff
@@ -0,0 +1,101 @@
+---
+title: Adapter Writing Guide
+description: This guide provides a step-by-step approach to creating adapters for the Truf Network (TN). Adapters facilitate the ingestion of data from various sources into TN, leveraging the SDK and Prefect for efficient data management.
+---
+
+> **Note**: While this guide uses Python for examples, adapters can also be written in other languages like Go or JavaScript. Refer to the respective SDKs for more information:
```
Comment (outerlook) [inline writing-adapters.mdx:6]: although the comment is true that we can insert data from other SDK, maybe the using the term

> adapters can also be written in another languages

could be misleading...

it's true people can build other frameworks in other languages, but maybe being more literal about is better? or am I off on some context?


Suggestion if we want to be more literal:
```suggestion
> **Note**: While adapters use a Python Framework, it's also possible to push data from other languages like Go or JavaScript. Refer to the respective SDKs for more information:
```

or of course any other way to make it more clear.

Comment (MicBun) [reply writing-adapters.mdx]: Thank you for the suggestion!

**Review by outerlook — APPROVED** [review]
all great

**Pull request merged**